apiary.apib

```diff
@@ -2910,6 +2910,77 @@ Update a milestone.
 
 + Response 204 (application/json;charset=utf-8)
 
+## Project items [/projectItems]
+
+Project items are added to a project and they affect the profitability. This information is only available for account administrators and project decision makers.
+
+*Required scopes: `projects`*
+
+### projectItems.list [GET /projectItems.list]
+
+Get a list of project items.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + project_id: `082e6289-30c5-45ad-bcd0-190b02d21e81` (string, optional)
+        + page (Page, optional)
+        + sort (array, optional)
+            + (object)
+                + field (enum[string], required)
+                    + Members
+                        + occurred_at
+                        + milestone_name
+                        + unit_cost
+                        + unit_price
+                        + quantity
+                        + cost
+                        + price
+                + order (enum[string], optional)
+                    + Members
+                        + asc
+                        + desc
+            + Default
+                + (object)
+                    + field: `occurred_at`
+                    + order: `desc`
+
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `cfb4146d-06be-41f1-bb39-aa3c929c71dc` (string)
+                + project (object)
+                    + type: `project` (string)
+                    + id: `944534fb-15f1-4eea-aab1-82a427aa2d0d` (string)
+                + milestone (object, nullable)
+                    + type: `milestone` (string)
+                    + id: `944534fb-15f1-4eea-aab1-82a427aa2d0d` (string)
+                + type: `tracked_time` (enum[string])
+                    + Members
+                        + tracked_time
+                        + external_cost_item
+                + description: `Drafted the document` (string, nullable)
+                + occurred_at: `2018-11-26T16:44:33+00:00` (string)
+                + participant (object, nullable)
+                    + type: `user` (string)
+                    + id: `a2d6bc44-2cf6-01c5-9a3c-67cabc1f029b` (string)
+                + unit_internal_cost (object)
+                    + Include Money
+                + unit_billable_price (object)
```

Comment:
Q: Currently, this naming is a bit weird and I reckon it would be much more readable to switch around some words. Should we call this `internal_unit_cost` instead? I think this would make more sense because we describe a cost per unit as `unit_cost`

/cc @stivni @jenssegers

Reply:
Discussed this with @stivni and @adognameddog and decided to use "internal_cost_per_unit" and "billable_price_per_unit"